products/BellHybrid/services/db/agents/QuotesAgent.cpp

```diff
@@ -42,4 +50,31 @@ namespace Quotes
         }
         return nullptr;
     }
+
+    auto QuotesAgent::handleGroupChanged(std::shared_ptr<db::Query> query) -> std::unique_ptr<db::QueryResult>
+    {
+        const auto request = std::dynamic_pointer_cast<Messages::InformGroupChanged>(query);
+        if (request) {
```

Comment:
I'd use early return pattern here, as this method has become quite heavily indented

Reply:
done